universe/wrappers/__init__.py

```diff
@@ -15,6 +15,7 @@
 from universe.wrappers.vectorize import Vectorize, Unvectorize, WeakUnvectorize
 from universe.wrappers.vision import Vision
 from universe.wrappers.recording import Recording
+from universe.wrappers.monitoring import Monitored
```

Comment:
Ah, that's a good idea, it just didn't occur to me. I'll give it a try. I think it'll allow getting rid of some of the weakref stuff, which is nice.